docs/user-guide/ze-usage.md

```diff
@@ -148,6 +149,15 @@ Your PDS member (or PS) is uploaded.
 
 <img src="../images/ze/ZE-mult-profiles.gif" alt="Add Profile">
 
+#### Allocate Like
+
```

Comment:
So maybe just a description below the title like:

"Select a data set and create a copy with the same parameters."

Reply:
That's the whole idea - we keep things short. I think developers appreciate this way more. Maybe, @jellypuno can speak to it

Reply:
Or Duplicate a dataset?

Reply:
Also, I do include the description of the feature. Line 44